MuvrKitTests/Resources/arms_model.layers.txt

```diff
@@ -0,0 +1 @@
+1200 id 250 relu 100 relu 3 logistic
```

Comment:
Are you sure they are correct? (Since in the analytics pr you used `tanh` for the 250 nodes layer)